driftbase/api/matches.py

```diff
@@ -40,6 +40,7 @@ class ActiveMatchesGetQuerySchema(ma.Schema):
     realm = ma.fields.String()
     version = ma.fields.String()
     player_id = ma.fields.List(ma.fields.Integer(), load_default=[])
+    match_id = ma.fields.List(ma.fields.Integer(), load_default=[])
```

Comment:
Are you querying on one match or several? Currently this is a list, but if this is for rejoining matches, you would normally only need one. If there are other use cases, that's fine, just want to know if it's intended or not.

Reply:
Since this can return multiple matches I kept it a list to maintain consistency with player_id and provide flexibility for other use cases. I'm only querying a single match_id in my case.